Assets/Scripts/Controllers/CharacterController.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class CharacterController : MonoBehaviour
+{
+    private Rigidbody2D _rb;
+    private Vector2 _movement;
+
+    [SerializeField] private float _movementSpeed = 500f;
+    [SerializeField] private float _jumpForce = 450f;
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        _rb = GetComponent<Rigidbody2D>();
+    }
+
+    private void FixedUpdate()
+    {
+        _rb.AddForce(_movement * _movementSpeed * Time.deltaTime);
```

Comment:
Note: You'll want to use `Time.fixedDeltaTime` to get the correct time-step adjustor while inside the `FixedUpdate` method. `deltaTime` should only really be used inside `Update` ticks.